R/col_types.R

```diff
@@ -360,9 +360,13 @@ vroom_select <- function(x, col_select, id) {
   # reorder and rename columns
   if (inherits(col_select, "quosures") || !rlang::quo_is_null(col_select)) {
     if (inherits(col_select, "quosures")) {
-      vars <- tidyselect::vars_select(c(id, names(spec(x)$cols)), !!!col_select)
+      vars <- tidyselect::vars_select(c(names(spec(x)$cols), id), !!!col_select)
```

Comment:
Using `.include` caused `id` to always be at the front. It looks like this was fixed for `tidyselect::eval_select()` https://github.com/r-lib/tidyselect/issues/224 which we want to switch to using later anyway. I don't think having id at the front is a problem when `id !%in% col_select` but otherwise I think it would be surprise if the columns are not in the same order as `col_select`.  What do you think?

Reply:
Yes, I'm happy to proceed with the existing fixup while we're still using `vars_select()` and not worry about `.include` at this time.